tests/ui/conftest.py

```diff
@@ -58,3 +60,33 @@ def user(base_url, fxa_account, jwt_token):
     assert requests.codes.created == r.status_code
     user.update(r.json())
     return user
+
+
+@pytest.yield_fixture(scope='session')
+def firefox_path(tmpdir_factory):
```

Comment:
I've realised that this will override any `--firefox-path` specified on the command line, and I'm wondering if that's really something we want, as it could be confusing and devs may want to test against specific versions of Firefox. Perhaps we should include `firefox_path` in the method signature, and if it's not None return it straight away.